src/plugin-slots/StudioFooterHelpButtonSlot/index.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { PluginSlot } from '@openedx/frontend-plugin-framework';
+import PropTypes from 'prop-types';
+import HelpButton from '../../components/studio-footer/help-components/HelpButton';
+
+const StudioFooterHelpButtonSlot = ({ isOpen, setIsOpen }) => (
+  <PluginSlot id="org.openedx.frontend.layout.studio_footer_help_button.v1" idAliases={['studio_footer_help_button_slot']} pluginProps={{ isOpen, setIsOpen }}>
```

Comment:
Since this is a new slot it shouldn't have `idAliases`

```suggestion
  <PluginSlot id="org.openedx.frontend.layout.studio_footer_help_button.v1" pluginProps={{ isOpen, setIsOpen }}>
```